### DISCUSSION TITLE
Flux  unable to update the git manifest file with the new release tag

### DISCUSSION THREAD

**Discussion by nkeng180** [fluxcd/flux2#4354] [Q&A]
I always have this issue, sometimes Flux doesn't pick up  release tag/version from our remote repo Nexus automatically to our git repo where the manifest files are. I have to manually go in  sometimes to update our deployment config file with the new build tag or i have to restart flux pod.  And helm operator sometimes will  not pick up the new release from git to apply on our k8s, i have to manually do that sometimes or restart helm operator pods.

Please i will appreciate any solution,

**Comment by kingdonb** [comment]
Are you using Flux v1 and Helm Operator? Please check out the migration docs:

https://fluxcd.io/flux/migration/

Flux v1 is not supported since 2021, its development has been stopped since Flux v2 has long ago passed the feature parity milestone.

If you need help with the migration, please join us on the [CNCF #flux slack](https://cloud-native.slack.com/channels/flux) channel, there are plenty of people who help as volunteers and we offer free migration workshops. It is a bit of learning curve to switch from Flux v1 to Flux v2; I used to maintain Flux v1, I am personally well versed in these issues, but if you haven't tried migrating then I'm not sure what issues you will encounter yet, other than to suggest to take a look at the migration docs from above.

If you can provide more information, then perhaps we can help!
